### DISCUSSION TITLE
How to automatically bypass sensors?

### DISCUSSION THREAD

**Discussion by Kipmo** [tsightler/ring-mqtt#381] [Support Q&A]
I have changed the state of "select.back_door_bypass_mode" to "Faulted" and then opened my back door and attempted to "alarm_arm_home" but Ring starts complaining that a sensor requires a bypass. When I then go check the state of select.back_door_bypass_mode I see that it was changed back to "Never" when the arming was attempted. What am I doing wrong?

**Comment by Kipmo** [comment]
I just tested it again and now it's working... the only difference was that I changed the state of the bypass sensor via the entities UI instead of via the Developer Options. This may have just been user error on my part.

**Comment by calisro** [comment]
https://www.home-assistant.io/docs/tools/dev-tools/#states

Setting it that way only changes the state in HA.  It doesn't actually talk to the remote device.  Its strictly temporary for testing something based on a state/attribute change really.
